lib/taskana-core/src/main/java/pro/taskana/spi/task/internal/BeforeRequestReviewManager.java

```diff
@@ -0,0 +1,50 @@
+package pro.taskana.spi.task.internal;
+
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.api.exceptions.SystemException;
+import pro.taskana.common.internal.util.SpiLoader;
+import pro.taskana.spi.task.api.BeforeRequestReviewProvider;
+import pro.taskana.task.api.models.Task;
+
+public class BeforeRequestReviewManager {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(BeforeRequestReviewManager.class);
+
+  private final List<BeforeRequestReviewProvider> beforeRequestReviewProviders;
+
+  public BeforeRequestReviewManager(TaskanaEngine taskanaEngine) {
+    beforeRequestReviewProviders = SpiLoader.load(BeforeRequestReviewProvider.class);
+    for (BeforeRequestReviewProvider serviceProvider : beforeRequestReviewProviders) {
+      serviceProvider.initialize(taskanaEngine);
+      LOGGER.info(
+          "Registered BeforeRequestReview service provider: {}",
```

Comment:
Why do you use "BeforeRequestReview service provider" here, but "BeforeRequestReviewProvider service provider" below? Which do we prefer?

Reply:
let's use the latter for consistency